zap/src/main/java/org/parosproxy/paros/network/HttpSender.java

```diff
@@ -259,6 +262,24 @@ private void checkState() {
         }
     }
 
+    private void updateCookieState() {
```

Comment:
Might worth merging this method with `checkState` so that we have only one method changing/setting up the state.